src/assets/room_devices/pet_bowl/pet_bowl.asset.ts

```diff
@@ -0,0 +1,47 @@
+DefineRoomDeviceAsset({
+	name: 'Pet bowl',
```

Comment:
The other asset names are typically starting every word of an asset name with a capital letter.